src/components/MARCTable/MARCTableRow.js

```diff
@@ -0,0 +1,446 @@
+        data-test-marc-table-in2
+        className={css.tableCell}
+        style={cellStyle}
+      >
+        <TextField
+          value={field.indicator2 || ''}
+          marginBottom0
+        />
+      </div>
+    );
+  };
+  const renderSubfieldField = () => {
+    const cellStyle = { width: columnWidths.subfield };
+
+    return (
+      <div
+        data-test-marc-table-cell
+        data-test-marc-table-subfield
+        className={css.tableCell}
+        style={cellStyle}
+      >
+        <TextField
+          value={field.subfield || ''}
+          marginBottom0
+        />
+      </div>
+    );
+  };
+  const renderSubactionField = () => {
+    const getMatchedSubaction = option => rowSubactions.some(subaction => subaction === option.value);
+
+    const cellStyle = { width: columnWidths.subaction };
+    const allowedOptions = SUBACTION_OPTIONS.filter(getMatchedSubaction);
+    const dataOptions = allowedOptions.map(option => ({
+      value: option.value,
+      label: intl.formatMessage({ id: option.label }),
+    }));
+
+    return (
+      <div
+        data-test-marc-table-cell
+        data-test-marc-table-subaction
+        className={css.tableCell}
+        style={cellStyle}
+      >
+        {!isEmpty(dataOptions) && (
+          <Select
+            dataOptions={dataOptions}
+            value={subactionValue || ''}
+            onChange={e => onSelectChange(e, setSubactionValue)}
+            placeholder={intl.formatMessage({ id: 'ui-data-import.settings.mappingProfile.marcTable.placeholder.select' })}
+            marginBottom0
+          />
+        )}
+      </div>
+    );
+  };
+  const renderDataField = () => {
+    const EDIT_ACTION = 'EDIT';
+    const MOVE_ACTION = 'MOVE';
+    const REPLACE_SUBACTION = 'REPLACE';
+    const NEW_SUBACTION = 'NEW';
+    const EXISTING_SUBACTION = 'EXISTING';
+
+    let cellStyle = {
+      width: columnWidths.data,
+      flexGrow: 1,
+      justifyContent: 'space-between',
+    };
+
+    const getContent = () => {
+      if (actionValue === EDIT_ACTION && subactionValue === REPLACE_SUBACTION) {
+        cellStyle = {
+          ...cellStyle,
+          display: 'flex',
+          'flex-direction': 'column',
+        };
+        const labelStyle = {
+          marginRight: '10px',
+          fontWeight: '400',
+          whiteSpace: 'nowrap',
+        };
+
+        const findContainerStyle = {
+          display: 'flex',
+          marginBottom: '10px',
+        };
+
+        return (
+          <div style={cellStyle}>
+            <div
+              data-test-marc-table-data-find
+              style={findContainerStyle}
+            >
+              <Label style={labelStyle}>
+                <FormattedMessage id="ui-data-import.settings.mappingProfile.marcTable.data.find" />
+              </Label>
+              <TextArea
+                value={dataValue?.find || ''}
+                onChange={e => onDataChange(e, 'find')}
+                marginBottom0
```

Comment:
Get a warning for this prop. Please check the other places.

Warning: React does not recognize the `marginBottom0` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `marginbottom0` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Reply:
Hm this prop is quite common in stripes components - https://github.com/folio-org/stripes-components/blob/master/lib/TextArea/TextArea.js#L25

Reply:
It seems like the issue with TextArea component itself.